core/modules/image/tests/image.test

```diff
@@ -265,13 +293,13 @@ class ImageStylesPathAndUrlUnitTest extends BackdropWebTestCase {
 
       // Repeat this with a different file that we do not have access to and
       // make sure that access is denied.
-      $file_noaccess = array_shift($files);
-      $original_uri_noaccess = file_unmanaged_copy($file_noaccess->uri, $scheme . '://', FILE_EXISTS_RENAME);
-      $generated_uri_noaccess = $scheme . '://styles/' . $this->style_name . '/' . $scheme . '/'. backdrop_basename($original_uri_noaccess);
-      $this->assertFalse(file_exists($generated_uri_noaccess), 'Generated file does not exist.');
-      $generate_url_noaccess = image_style_url($this->style_name, $original_uri_noaccess);
+      $file_no_access = array_shift($files);
+      $original_uri_no_access = file_unmanaged_copy($file_no_access->uri, $scheme . '://', FILE_EXISTS_RENAME);
+      $generated_uri_no_access = $scheme . '://styles/' . $this->style_name . '/' . $scheme . '/'. backdrop_basename($original_uri_no_access);
+      $this->assertFalse(file_exists($generated_uri_no_access), 'Generated file does not exist.');
+      $generate_url_no_access = image_style_url($this->style_name, $original_uri_no_access);
 
-      $this->backdropGet($generate_url_noaccess);
+      $this->backdropGet($generate_url_no_access);
```

Comment:
Love this cleanup here ...assuming that CSpell complained about the `noaccess` word.